schemas/layouts/layouts.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
meta: why's this file `layouts.json` rather than `layout.json`? Your other files are singular rather than plural.